### PR TITLE
Add test case show casing issue 92

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

File: t/assets/sass/themed-bootstrap.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@import "../cache/raw.githubusercontent.com/twbs/bootstrap-sass/a73cc0f0e5c794206e9a70bc0b67e67cf37c1bca/assets/stylesheets/_bootstrap.scss";
+
+h1 { color: red; }
```
Comment (jhthorsen) [inline t/assets/sass/themed-bootstrap.scss]: You can jut put this in the `__DATA__` section of the unit test instead. Look at the other tests for examples.

**Comment by jhthorsen** [comment]
What are you trying to show with this test?

**Comment by coolo** [comment]
the fonts in app.css are not fetched and referenced as ../fonts and not as expected as ../assets/XXXX/ - they are just sassed.

File: t/assets/sass/themed-bootstrap.scss (hunk 1 above)
Comment (coolo) [inline t/assets/sass/themed-bootstrap.scss]: sorry if I made the impression I wanted to be an assetpack developer ;)

**Comment by jhthorsen** [comment]
Could you make the test fail?

**Comment by coolo** [comment]
now that's an interesting one - because I get http://paste.opensuse.org/82710413

**Comment by jhthorsen** [comment]
What does `mojo version` say? Do you have IO::Socket::SSL installed? What about proxy settings..? Are you behind a proxy?

**Comment by coolo** [comment]
coolo@gertrude#openQA>mojo version
CORE
  Perl        (v5.24.0, linux)
  Mojolicious (7.05, Doughnut)

OPTIONAL
  EV 4.0+                 (n/a)
  IO::Socket::Socks 0.64+ (n/a)
  IO::Socket::SSL 1.94+   (2.025)
  Net::DNS::Native 0.15+  (n/a)

You might want to update your Mojolicious to 7.06!

No proxy. And I wonder how it would relate to the rewriting of the URLs.

**Comment by coolo** [comment]
For people falling over the same: The workaround in #92 still works fine:

Have bootstrap css registered as << in the .def - which fetches the fonts just fine. And then have $icon-font-path: '/fonts/'; in your sass theme, so that sass generates /fonts/... instead ../fonts.

And then you can symlink from /public like this:
```
coolo@gertrude#openQA>ls -l public/fonts/
insgesamt 16
lrwxrwxrwx 1 coolo suse 97  6. Sep 15:04 glyphicons-halflings-regular.eot -> ../../assets/cache/maxcdn.bootstrapcdn.com/bootstrap/3.3.6/fonts/glyphicons-halflings-regular.eot
lrwxrwxrwx 1 coolo suse 97  6. Sep 15:04 glyphicons-halflings-regular.ttf -> ../../assets/cache/maxcdn.bootstrapcdn.com/bootstrap/3.3.6/fonts/glyphicons-halflings-regular.ttf
lrwxrwxrwx 1 coolo suse 98  6. Sep 15:04 glyphicons-halflings-regular.woff -> ../../assets/cache/maxcdn.bootstrapcdn.com/bootstrap/3.3.6/fonts/glyphicons-halflings-regular.woff
lrwxrwxrwx 1 coolo suse 99  6. Sep 15:04 glyphicons-halflings-regular.woff2 -> ../../assets/cache/maxcdn.bootstrapcdn.com/bootstrap/3.3.6/fonts/glyphicons-halflings-regular.woff2
```

**Comment by jhthorsen** [comment]
Closing this because of inactivity.

**Pull request closed without merging**